tests/test_citations.py

```diff
@@ -149,8 +149,8 @@ def _parsed_citation_styles(self):
         # populate the DB with parsed citation styles
         try:
             parse_citation_styles.main()
-        except OSError:
-            pass
+        except OSError as e:
+            raise e
```

Comment:
This is to have better visibility why tests are failing:
In my case they failed locally because I didn't have citation style files, and I spent many more time  to debug this issue than I could have if this change was present